lib/l10n/intl_es.arb

```
@@ -686,5 +686,64 @@
+  },
+  "notificationSendInvoice": "Por favor envía una factura",
+  "notificationSendInvoiceAmount": "Por favor envía una factura por {amount} satoshis",
+  "@notificationSendInvoiceAmount": {
+    "placeholders": {
+      "amount": {
+        "type": "String"
+      }
+    }
+  },
+  "notificationFiatSent": "El pago fiat ha sido enviado. Esperando confirmación.",
+  "notificationBitcoinReleased": "¡Bitcoin ha sido liberado! Tu factura debería ser pagada pronto.",
```

RuboCop (1.76.1)

[warning] Duplicated key in hash literal.

(Lint/DuplicateHashKey)